tests/integration/conftest.py

```diff
@@ -45,6 +45,7 @@
 # Use the helpers folder to store test helpers.
 # See: http://stackoverflow.com/a/33515264/374865
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'helpers'))
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.dirname(__file__)), 'test_factory'))
```

Comment:
The test dir is not a module (for valid reasons...) thus we need this trick to add the test factory dir to the system path.

Reply:
Alternative idea: put it in `tests/helpers/factories/db`.

Reply:
Yesss

Reply:
done